package.json

```diff
@@ -17,6 +17,7 @@
     "build:cjs": "tsc -p tsconfig.cjs.json",
     "prepublishOnly": "npm run build",
     "dev": "tsc --watch -p tsconfig.esm.json",
+    "lint": "eslint .",
     "test": "eslint . && jest",
```

Comment:
we can probably remove eslint from the test command since it's a separate workflow now

Reply:
also fine to keep but I'd rather not run the linting twice 🤷